python/lsst/sims/featureScheduler/basis_functions.py

```diff
@@ -501,6 +505,74 @@ def __call__(self, indx=None):
 
         return result
 
+class Normalized_Target_map_basis_function(Base_basis_function):
+    """Normalize the maps first to make things smoother
+    """
+    def __init__(self, filtername='r', nside=default_nside, target_map=None,
+                 survey_features=None, condition_features=None, norm_factor=240./2.5e6,
+                 out_of_bounds_val=-10.):
+        """
+        Parameters
+        ----------
+        filtername: (string 'r')
+            The name of the filter for this target map.
+        nside: int (default_nside)
+            The healpix resolution.
+        target_map : numpy array (None)
+            A healpix map showing the ratio of observations desired for all points on the sky
+        norm_factor : float (800./2.5e6)
+            for converting target map to number of observations. This is a convience scaling,
+            It should be degenerate with the weight of the basis function.
+        out_of_bounds_val : float (10.)
+            Point value to give regions where there are no observations requested
+        """
+        if nside is None:
+            nside = utils.set_default_nside()
+
+        self.norm_factor = norm_factor
+        if survey_features is None:
+            self.survey_features = {}
+            # Map of the number of observations in filter
+            self.survey_features['N_obs'] = features.N_observations(filtername=filtername)
+            # Count of all the observations
+            self.survey_features['N_obs_count_all'] = features.N_obs_count(filtername=None)
+        super(Target_map_basis_function, self).__init__(survey_features=self.survey_features,
```

Comment:
I believe this should be `super(Normalized_Target_map_basis_function, self)` instead of `super(Target_map_basis_function, self)`.